components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -55,11 +62,27 @@ class MozillaSocorroService(
     private val version: String = BuildConfig.MOZILLA_VERSION,
     private val buildId: String = BuildConfig.MOZ_APP_BUILDID,
     private val vendor: String = BuildConfig.MOZ_APP_VENDOR,
-    private val serverUrl: String = "https://crash-reports.mozilla.com/submit?id=$appId&version=$version&$buildId"
+    private var serverUrl: String? = null,
+    private var versionName: String = "0.0"
```

Comment:
Why can this get passed in if we override it in `init()`?

Reply:
I gave it a default value for the case when the caller didn't pass in a value or we failed to get the `PackageInfo`.
This way we will at least have "0.0" as the value.

Reply:
After some thoughts, I've updated this to default as "N/A".  This makes more sense when we fails to (shouldn't happen) retrieve `PackageInfo`.